src/Worker.Extensions.DurableTask.Analyzers/Worker.Extensions.DurableTask.Analyzers.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
```

Comment:
as noted elsewhere, the indentation is a bit large - can it be more conservative?

Reply:
Good catch haha definitely!
I used VS to format it, but for some reason it used tabs over spaces, I am going to fix it. Thanks